rpc/src/v1/helpers/dispatch.rs

```diff
@@ -474,7 +474,7 @@ pub fn execute<D: Dispatcher + 'static>(
 					.map(ConfirmationResponse::SignTransaction)
 				).boxed()
 		},
-		ConfirmationPayload::Signature(address, mut data) => {
+		ConfirmationPayload::PostSignTransaction(address, mut data) => {
```

Comment:
+1 - definitely shouldn't be called `PostSignTransaction` - it's definitely not signing a transaction of any variety.